java-frontend/src/main/java/org/sonar/java/se/xproc/MethodBehavior.java

```diff
@@ -102,11 +120,11 @@ public int methodArity() {
   }
 
   private boolean isVoidMethod() {
-    return methodSymbol.returnType().type().isVoid();
+    return methodSymbol == null ? org.objectweb.asm.Type.getReturnType(signature.substring(signature.indexOf('('))) == org.objectweb.asm.Type.VOID_TYPE : methodSymbol.returnType().type().isVoid();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS864)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 196 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)